app/code/Magento/CatalogInventory/Api/InventoryRegistryInterface.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Magento\CatalogInventory\Api;
+
+use Magento\CatalogInventory\Api\Data\InventoryInterface;
+use Magento\CatalogInventory\Api\Data\InventoryRecordInterface;
+
+/**
+ * Inventory registry interface
+ *
+ * The main point, where inventory information is retrieved for Magento
+ *
+ * @api
+ */
+interface InventoryRegistryInterface
+{
+    /**
+     * Returns main store inventory based on store configuration
+     * or other environment dependencies
+     *
+     * This one should be used only for filtering stock status on collections,
+     * used in indexers for applying inventory status limitations
+     *
+     * @param int|null $storeId
+     * @return InventoryInterface
+     */
+    public function getInventory($storeId = null);
```

Comment:
Yes, as Inventory now do not have store id binded to it, but hovewer it should be possible to specify default inventory for a store, that's why there is option store parameter specified for it. Take a look at this chart to get the idea:
![Usage of Intentory Registry](https://www.lucidchart.com/publicSegments/view/55e5dcd3-8174-4289-9c64-71df0a00519d/image.png)